content/concepts/introduction.md

```diff
@@ -3,20 +3,23 @@ title: Introduction
 description: Ocean Protocol - Tools for the Web3 Data Economy
 ---
 
-In Ocean Protocol, each data service gets its own ERC20 **datatoken**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
+In Ocean Protocol, each data service gets its own ERC721 **data NFT** and one or more ERC20 **datatoken**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
 
 OCEAN token is used for staking, and more. [Here](https://oceanprotocol.com/token) are details.
 
-Ocean Protocol provides tools for developers to _build data markets_, and to _manage datatokens_ for use in DeFi.
+Ocean Protocol provides tools for developers to _build data markets_, and to _manage data NFTs and datatokens_ for use in DeFi.
 
 **Build Data Markets.** Use Ocean Protocol software tools to build your own data marketplace, by either forking [Ocean Market](https://market.oceanprotocol.com/) code or building up with Ocean components.
 
-**Manage Datatokens for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean.js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage datatokens:
+**Manage Datatokens and Data NFTs for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean.js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage data NFTs and datatokens:
 
-- _Publish and consume data services:_ downloadable files or compute-to-data. Use Ocean to deploy a new [ERC20](https://github.com/ethereum/EIPs/blob/7f4f0377730f5fc266824084188cc17cf246932e/EIPS/eip-20.md) datatoken contract for each data service, then mint datatokens.
+- _Publish and consume data services:_ downloadable files or compute-to-data. Use Ocean to deploy a new [ERC721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md and [ERC20](https://github.com/ethereum/EIPs/blob/7f4f0377730f5fc266824084188cc17cf246932e/EIPS/eip-20.md) datatoken contract for each data service, then mint datatokens.
```

Comment:
Remove the space between [ERC721] and the link.
The link and the displayed text are not formatted properly 
![Screenshot 2022-03-11 at 14 19 42](https://user-images.githubusercontent.com/84312885/157865994-6e15a083-8ba5-4063-9c53-a487a9f6248a.png)

Reply:
done